frontend/src/components/ScheduledEventsRunHistoryTableRow/utils/index.ts

```diff
@@ -0,0 +1,28 @@
+const status: { [char: string]: string } = {
+  p: 'Pending',
+  r: 'Running',
+  c: 'Completed',
+  e: 'Errored',
+};
+
+const statusClasses: { [char: string]: string } = {
+  p: 'badge-secondary',
+  r: 'badge-warning',
+  c: 'badge-success',
+  e: 'badge-danger',
+};
+export const convertStatus = (statusValue: string): string | undefined => {
+  for (const key in status) {
```

Comment:
Do you have a system on when to have a blank line or is it just random? I prefer consistency ...